lsp-json.el

```diff
@@ -0,0 +1,136 @@
+;;; lsp-pwsh.el ---  vscode-json-languageserver integration -*- lexical-binding: t; -*-
```

Comment:
this filename in this comment is wrong

Reply:
Oh, the downfall of copy/paste :(. Will update it next time I touch this file again.

Reply:
No worries.